kclvm/sema/src/resolver/import.rs

```diff
@@ -59,7 +59,7 @@ impl<'ctx> Resolver<'ctx> {
                                         pos: Position {
                                             filename: self.ctx.filename.clone(),
                                             line: stmt.line,
-                                            column: Some(1),
+                                            column: None,
```

Comment:
It can be modified in another Issue and PR.

Reply:
fixed, pr#128